Example/Custom Integration/FPXExampleViewController.m

```diff
@@ -86,9 +86,14 @@ - (void)updateUIForPaymentInProgress:(BOOL)paymentInProgress {
     }
 }
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated"
-- (void)pay {
+- (void)selectBank {
+    STPBankSelectionViewController *vc = [[STPBankSelectionViewController alloc] initWithBankType:STPBankTypeFPX];
+    vc.delegate = self;
+    
+    [self.navigationController pushViewController:vc animated:YES];
+}
+
+- (void)payWithBankAccount:(STPFPXBankBrand)bank {
```

Comment:
Refactor to take a `STPPaymentMethodParams` instead?

Reply:
Hah, oops